cepy/build_cephes.py

```diff
@@ -0,0 +1,134 @@
+# Copyright 2017 Gamalon, Inc.
```

Comment:
Let's keep the source code for the Cephes library all in a single directory with no additional files. This way the Cephes distribution can just be plopped into a single directory if it needs to be updated, and it is clear from a copyright perspective that it is a single cohesive piece of software that has not been modified.

Reply:
Thomas, the suggestion I made here for the directory structure still holds.